### PR TITLE
Always look in standard library location

### PR DESCRIPTION
**Pull request by mavit**
This location isnʼt mentioned in libraryfolders.vdf.

### CODE REVIEW

File: steam_desktop_updater.py [hunk 1]
```diff
@@ -163,7 +163,9 @@ def get_installed_apps(steam_root):
     apps = []
     logging.info('Searching library folders')
     with open(os.path.join(steam_root, 'steamapps', 'libraryfolders.vdf'), 'r') as lf:
-        for k, v in vdf.load(lf)['LibraryFolders'].items():
+        library_folders = vdf.load(lf)['LibraryFolders']
+        library_folders['0'] = steam_root
```
Comment (gasinvein) [inline steam_desktop_updater.py:168]: Pushing items into LibraryFolders doesn't seems safe, e.g. if Steam will add item `"0"` there - we'll end up overwriting it.

Comment (gasinvein) [reply steam_desktop_updater.py:168]: Can you at least add an assertion before assingning the value? E.g.
```suggestion
        assert '0' not in library_folders
        library_folders['0'] = steam_root
```

Comment (mavit) [reply steam_desktop_updater.py:168]: Let's handle that by building a new list.

Comment (gasinvein) [reply steam_desktop_updater.py:168]: Too late, already merged! But yeah, a new list would be better.

Comment (mavit) [reply steam_desktop_updater.py:168]: I see that you beat me to it, but there's an alternative solution at https://github.com/mavit/steam-desktop-updater/commit/807b43ae47e0a1c50a98e8495f3107e6e3ff90fa

Comment (gasinvein) [reply steam_desktop_updater.py:168]: Don't mind sending it as an another PR?

**Pull request merged**